assets/admin/editor-wizard/steps/patterns-step.js

```diff
@@ -56,12 +57,42 @@ const PatternsStep = ( { title, replaces, onCompletion } ) => {
 			<h1 className="sensei-editor-wizard-step__sticky-title">
 				{ title }
 			</h1>
+			<NoLayoutsWarning layouts={ availableTemplates } />
 			<Slot />
 			<PatternsList onChoose={ onChoose } />
 		</div>
 	);
 };
 
+/**
+ * Output a warning message if no layouts are available.
+ *
+ * @param {Object} props         Compoent props.
+ * @param {Object} props.layouts Available layouts.
+ */
+const NoLayoutsWarning = ( { layouts } ) => {
+	if ( layouts && Object.keys( layouts ).length > 0 ) {
+		return <></>;
+	}
+
+	/**
+	 * Filters the warning message when no layouts are available.
+	 *
```

Comment:
@donnapep Couldn't add `@hook` here. Is it expected?

![CleanShot 2024-03-07 at 21 45 56@2x](https://github.com/Automattic/sensei/assets/329356/d802f93e-6eb0-46d2-8a6f-ce915aa10121)